front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -49,6 +50,8 @@ import Link from 'utils/cl-router/Link';
 import eventEmitter from 'utils/eventEmitter';
 import { convertUrlToUploadFile, isUploadFile } from 'utils/fileUtils';
 import { isNilOrError } from 'utils/helperUtils';
+import { isAdmin } from 'utils/permissions/roles';
```

Comment:
We don't use these utils and rules directly. Instead, there is a react hook `usePermission` which integrates them and should give you the answer to whether something is permitted or not. If the specific rule doesn't exist, you should create it.

Example from comments (you can find many others):
``` 
const canDelete = usePermission({
    item: comment,
    action: 'delete',
    context: { projectId },
  });
```